src/Common/tests/System/Net/Capability.Security.cs

```diff
@@ -2,13 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.InteropServices;
 using System.Security.Cryptography.X509Certificates;
 using System.Threading;
 
 namespace System.Net.Test.Common
 {
     public static partial class Capability
     {
+        // Curl + OSX SecureTransport doesn't support the custom certificate callback.
+        public static bool BackendSupportsCustomCertificateHandling =>
+#if TargetsWindows
```

Comment:
We try to avoid using any ifdef's like this.  So, perhaps you can refactor this out to a separate file and use partial classes with conditional compilation in the *.csproj file.